system_tests/state_fuzz_test.go

```diff
@@ -131,10 +131,22 @@ func (c noopChainContext) Engine() consensus.Engine {
 	return nil
 }
 
+func (c noopChainContext) CurrentHeader() *types.Header {
+	return nil
+}
+
 func (c noopChainContext) GetHeader(common.Hash, uint64) *types.Header {
 	return nil
 }
 
+func (c noopChainContext) GetHeaderByNumber(uint64) *types.Header {
+	return nil
+}
+
+func (c noopChainContext) GetHeaderByHash(common.Hash) *types.Header {
+	return nil
+}
+
```

Comment:
why add these given they aren't used in the upstream commit? https://github.com/OffchainLabs/nitro/blob/v3.9.0/system_tests/state_fuzz_test.go#L121-L136